onken/workspace/views.py

```diff
@@ -1,5 +1,16 @@
 from django.http import HttpResponse
+from django.contrib.auth.decorators import login_required
+import django_cas_ng.views
+from django.http import HttpResponseForbidden
 
 
+@login_required
 def index(request):
     return HttpResponse("This is the workspace app. This workspace is %s." % request.tenant.name)
+
+
+def login(request):
+    if request.user.is_authenticated and not request.user.is_staff:
+        return HttpResponseForbidden()
+    else:
+        return django_cas_ng.views.login(request)
```

Comment:
Instead of redeclaring this view here, please re-use the one in the public module.

Reply:
Can you do that?

Reply:
actually I'm dumb this is for admin which is getting yanked anyway

Reply:
do remove this view though